### PR TITLE
Web4542 update coding styles

### PR DESCRIPTION
**Pull request by lukefrake**
#### What are the links to relevant tickets?
WEB-4542
#### What does this PR do?
Add make-up
Make pass hinting linting etc
#### What unit tests does this PR have?
N/A
#### How should a developer review this?
Pull down & run tests / see travis run
#### Any background context you want to provide?
Part of something bigger
#### Screenshots (if appropriate)
---
#### Quality checklist (for PR author to complete BEFORE code review):
- [x] I've checked there is appropriate unit test coverage.
- [x] I've updated documentation with any changes to procedures.
- [x] I've tested this cross browser for any visual changes.
- [x] I've checked this work against the requirements of the Jira.
- [x] I am ready for this to be code reviewed, merged and tested.

#### Reviewer 1
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve witnessed the tests running succesfully.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

#### Reviewer 2
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve witnessed the tests running succesfully.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

### CODE REVIEW

**Comment by philmeehan** [comment]
I'll take reviewer 1

File: package.json [hunk 1]
```diff
@@ -16,18 +16,24 @@
...
-    "grunt": ".4.4",
-    "grunt-contrib-jshint": "0.10.0",
-    "grunt-jscs": "0.6.2",
+    "coveralls": "2.11.1",
+    "grunt": "0.4.5",
+    "grunt-contrib-jshint": "0.11.0",
+    "grunt-jscs": "1.5.0",
     "grunt-mocha-test": "0.11.0",
     "hapi": "6.0.2",
+    "istanbul": "^0.3.7",
```
Comment (notABluesSinger) [inline package.json]: remove the ^

File: package.json [hunk 2]
```diff
@@ -16,18 +16,24 @@
...
     "lodash": "2.4.1",
+    "make-up": "^1.0.3",
```
Comment (notABluesSinger) [inline package.json]: remove the ^

**Comment by notABluesSinger** [comment]
I'll take reviewer 2

**Comment by lukefrake** [comment]
Thanks man, updated

**Pull request merged**